src/containers/explore/EntityNeighborsContainer.js

```diff
@@ -0,0 +1,177 @@
+import type { RequestState } from 'redux-reqseq';
+
+import EntityNeighborsTable from './EntityNeighborsTable';
+import { EXPLORE_ENTITY_NEIGHBORS } from './actions';
+
+import { DataSetTitle } from '../../components';
+import { EXPLORE } from '../../core/redux/constants';
+import { selectOrgDataSets } from '../../core/redux/selectors';
+
+const ContainerWrapper = styled.div`
+  min-height: 500px;
+`;
+
+const SectionWrapper = styled.section`
+  margin: 15px 0;
+`;
+
+type Props = {
+  isModal :boolean;
+  neighbors :Map;
+  organizationId :UUID;
+};
+
+const EntityNeighborsContainer = ({ isModal, neighbors, organizationId } :Props) => {
+  const [visibleOptions, setVisibleOptions] = useState(Map());
+  const [visibleNeighbors, setVisibleNeighbors] = useState(Set());
+  const exploreEntityNeighborsRS :?RequestState = useRequestState([EXPLORE, EXPLORE_ENTITY_NEIGHBORS]);
+
+  const associationEntitySetIds :Set<UUID> = useMemo(() => (
+    neighbors ? neighbors.keySeq().toSet() : Set()
+  ), [neighbors]);
+
+  const dataSetIds :Set<UUID> = useMemo(() => (
+    Set().withMutations((set) => {
+      if (neighbors) {
+        neighbors.reduce((ids :Set<UUID>, esNeighborsMap :Map) => ids.add(esNeighborsMap.keySeq()), set);
+      }
+    }).flatten()
+  ), [neighbors]);
+
+  const dataSetsMap :Map = useSelector(selectOrgDataSets(organizationId, dataSetIds));
+  const associationEntitySetsMap = useSelector(selectOrgDataSets(organizationId, associationEntitySetIds));
+
+  const handleAssociationOnClick = (event :SyntheticEvent<HTMLButtonElement>) => {
+    const { currentTarget } = event;
+    if (currentTarget instanceof HTMLElement) {
+      const { value: dataSetId } = currentTarget;
+      if (dataSetId) {
+        if (visibleOptions.get(dataSetId, Set()).isEmpty()) {
+          const neighborsDataSetIds :Set = neighbors.get(dataSetId, Map()).keySeq().toSet();
+          setVisibleOptions(visibleOptions.set(dataSetId, neighborsDataSetIds));
+        }
+        else {
+          setVisibleOptions(visibleOptions.delete(dataSetId));
+        }
+      }
+    }
+  };
+
+  const handleNeighborOnClick = (event :SyntheticEvent<HTMLButtonElement>) => {
+    const { currentTarget } = event;
+    if (currentTarget instanceof HTMLElement) {
+      const { value: entitySetId } = currentTarget;
+      if (visibleNeighbors.includes(entitySetId)) {
+        setVisibleNeighbors(visibleNeighbors.delete(entitySetId));
+      }
+      else {
+        setVisibleNeighbors(visibleNeighbors.add(entitySetId));
+      }
+    }
+  };
+
+  const associationEntitySetChips = associationEntitySetsMap.valueSeq().map((associationEntitySet :Map) => {
+    const associationEntitySetName = associationEntitySet.get('title') || associationEntitySet.get('name');
+    const associationEntitySetId = associationEntitySet.get('id');
+    return (
+      <Checkbox
+          checked={!visibleOptions.get(associationEntitySetId, Set()).isEmpty()}
+          label={associationEntitySetName}
```

Comment:
To fix this: `Warning: Each child in a list should have a unique "key" prop.`

```suggestion
          label={associationEntitySetName}
          key={associationEntitySetId}
```